examples/Billing/GetInvoices.php

```diff
@@ -0,0 +1,212 @@
+        ]);
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())
+            ->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID,
+                $options[ArgumentNames::BILLING_SETUP_ID] ?: self::BILLING_SETUP_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+            exit(1);
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+            exit(1);
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $billingSetupId the billing setup ID to filter the invoices on
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $billingSetupId
+    ) {
+        // Gets the date one month before now.
+        $lastMonth = strtotime('-1 month');
+
+        // [START GetInvoices]
+        // Issues the request.
+        $response = $googleAdsClient->getInvoiceServiceClient()->listInvoices(
+            $customerId,
+            ResourceNames::forBillingSetup($customerId, $billingSetupId),
+            // The year needs to be 2019 or later.
+            date('Y', $lastMonth),
+            MonthOfYear::value(strtoupper(date('F', $lastMonth)))
+        );
+        // [END GetInvoices]
+
+        // [START GetInvoices_1]
+        // Iterates over all invoices retrieved and prints their information.
+        foreach ($response->getInvoices() as $invoice) {
+            /** @var Invoice $invoice */
+            printf(
+                "- Found the invoice '%s':" . PHP_EOL .
+                "  ID (also known as Invoice Number): %s" . PHP_EOL .
```

Comment:
Is it a string like primary billing ID that has hyphens?
If so, could you add a single quote?
Please do the same for other IDs in this example.

Reply:
Done. I also did it for the fields that may have spaces in their values (e.g., names, descriptions).